Default Release Upload URL Fails

When #4 was merged, we started having the following failure with our release asset uploads:

```shell
> Run csexton/release-asset-action@master
Uploading file: bundles/adafruit-circuitpython-bundle-4.x-mpy-20200207.zip
Uploading file: bundles/adafruit-circuitpython-bundle-5.x-mpy-20200207.zip
Uploading file: bundles/adafruit-circuitpython-bundle-examples-20200207.zip
Uploading file: bundles/adafruit-circuitpython-bundle-py-20200207.zip
Uploading file: bundles/z-build_tools_version-1.4.1.ignore
##[error]Upload failed: Cookies must be enabled to use GitHub. undefined
Uploaded z-build_tools_version-1.4.1.ignore
##[error]Upload failed:  undefined
Uploaded adafruit-circuitpython-bundle-examples-20200207.zip
##[error]Upload failed:  undefined
Uploaded adafruit-circuitpython-bundle-4.x-mpy-20200207.zip
##[error]Upload failed:  undefined
Uploaded adafruit-circuitpython-bundle-py-20200207.zip
##[error]Upload failed:  undefined
Uploaded adafruit-circuitpython-bundle-5.x-mpy-20200207.zip
##[error]Node run failed with exit code 1
```

It took us a little while to figure out what would cause this, since the error messages didn't really point to the root cause. I finally settled on researching `Cookies must be enabled to use GitHub` and came across an old `octokit/rest.js` issue that pointed out the need to send requests to `https://api.github.com` vs `https://github.com`.

After debugging on a fork, I realized that this line was resulting in the default of a non-api URL:
[`url= core.getInput('release-url', {required: false}) || context.payload.release.html_url;`](https://github.com/csexton/release-asset-action/blob/master/main.js#L56)

By changing `|| context.payload.release.html_url` --> `|| context.payload.release.upload_url`, I had successful uploads.

I know you can now pass in the `release-url` variable, and I've worked that out using the available context expression in the workflow. However, we'd have over 200 repositories to update with that approach.

I'm just wanting to gauge interest, or reasoning, for having the default release upload URL be one that will fail.

I would happily submit a PR if a change is desired.

Also, thanks @csexton for this Action. Having glob pattern uploads is a life saver!

--- a/main.js
+++ b/main.js
@@ -53,7 +53,7 @@ async function run() {
         return;
       }
   }else{
-      url= core.getInput('release-url', {required: false}) || context.payload.release.html_url;
+      url= core.getInput('release-url', {required: false}) || context.payload.release.upload_url;
   }
 
   core.setOutput('url', url );

--- a/main.js
+++ b/main.js
@@ -58,6 +58,8 @@ async function run() {
 
   core.setOutput('url', url );
 
+  console.log(`Uploading release assets to: ${url}`)
+
   var list = [];
 
   // Add any single files

